Module_Postman/activities/Act2_Creating_and_Testing_BitBloxs/9.md

```diff
@@ -0,0 +1,21 @@
+# PUT Request
+
+If while you are sending your POST request, you receive
+
+```json
+{
+    "data": "You cannot send a POST request to an already colored tile. You have to use a PUT request."
+}
+```
+
+Then, it is because you are sending a POST request to an already colored tile. 
+
+In this activity, we will be using PUT request to update the color of boxes.
+
+According to documentation, the routes for the PUT request is the following: `https://bitbloxs.herokuapp.com/change/<int:box_id>/<string:color>`
+
+Go ahead, and set up a PUT request in Postman. 
+
+![1578940821931](C:\Users\caiqi\AppData\Roaming\Typora\typora-user-images\1578940821931.png)
```

Comment:
just a headsup that image file location needs fixed here, has a local file reference